CHANGELOG.md

```diff
@@ -51,7 +51,7 @@ All notable changes to this project will be documented in this file.
 -   #1744 : Add support for set method `intersection()`.
 -   #1745 : Add support for set method `intersection_update()`.
 -   #1745 : Add support for set method `isdisjoint()`.
--   #2059 : Add C support for returning dictionaries from functions.
+-   #2059 : Add C and Fortran support for returning dictionaries from functions.
```

Comment:
How do you decide if you should mention #2059 or #2277 here?

Reply:
I usually just leave whichever was mentioned first. The two issues are almost a copy/paste of one another